framework/test/unit/modules/token/cc_method.spec.ts

```diff
@@ -405,11 +454,73 @@ describe('TokenInteroperableMethod', () => {
 				escrowStore.getKey(sendingChainID, defaultTokenID),
 			);
 			expect(amount).toEqual(defaultEscrowAmount - fee);
-			const { amount: receiver } = await escrowStore.get(
+			const { availableBalance } = await userStore.get(
+				methodContext,
+				userStore.getKey(defaultAddress, defaultTokenID),
+			);
+			expect(availableBalance).toEqual(defaultAccount.availableBalance + fee);
+			checkEventResult(
+				methodContext.eventQueue,
+				BeforeCCMForwardingEvent,
+				TokenEventResult.SUCCESSFUL,
+			);
+		});
+
+		it('should deduct sending chain escrow account for fee and credit to relayer account if ccm did not fail', async () => {
```

Comment:
Test case seemingly validate if receiving chain is credited, if CCM didn't fail. Could be rephrased to 'should deduct sending chain escrow account for fee and credit to receiving chain account if ccm did not fail`?

Reply:
I've updated the logic to use `ccmFailed`, test description and updated comment in the source code to make it more clear